sim/midas/src/main/cc/simif.h

```diff
@@ -269,6 +201,11 @@ class simif_t {
   /// Return the name of the simulated target.
   std::string_view get_target_name() const { return config.target_name; }
```

Comment:
Random aside: In the spirit of further simplifying simif, in a future PR you may consider pushing this into the testing related classes.